assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-tiles.scss

```diff
@@ -31,29 +31,17 @@
 		border-radius: $br-sm;
 
 		.googlesitekit-widget-audience-tiles__tabs {
-			border-bottom: 1px solid $c-utility-divider;
```

Comment:
The tabs in the new design use the same styling as the Audience Tiles widget's tabs, so the common style has been extracted for reuse in the `googlesitekit-tab-bar--start-aligned-high-contrast` class below.